styleguide/_themes/derek/scss/patterns/grid/businessResource.scss

```diff
@@ -1,14 +1,24 @@
-.resource {
+$hover-color: 12px solid rgba(0, 0, 0, .4);
```

Comment:
Could this be used elsewhere? If not, why not set the property directly and skip the variable?